docs/templating/twig_function_reference/checkout_twig_functions.md

```diff
@@ -0,0 +1,77 @@
+---
+description: Checkout Twig return information about the checkout process, and total values related to cart and cart items.
+edition: commerce
+---
+
+# Checkout Twig functions
+
+You can use Checkout Twig functions to get information about the checkout process, 
+as well as total values related to cart and cart items.
+
+### `ibexa_checkout_step_label()`
+
+The 'ibexa_checkout_step_label()' function returns a name of the step (configured in `framework.workflows.workflow.ibexa_checkout.transitions.<transition>.metadata.label`).
+
+``` html+twig
+{% block title %}
+    <h1>{{ ibexa_checkout_step_label(checkout, step) }}</h1>
+{% endblock %}
+```
+
+### `ibexa_checkout_steps()`
+
+The `ibexa_checkout_steps()` function returns a list of steps configured in `framework.workflows.workflow.ibexa_checkout.transitions`).
+
+``` html+twig
+{% for step in ibexa_checkout_steps(checkout) %}
+    // ...
+{% endfor %}
+```
+
+### `ibexa_checkout_step_path()`
+
+The `ibexa_checkout_step_path()` function returns a path to the step.
+
+``` html+twig 
+<a href="{{ ibexa_checkout_step_path(checkout, step) }}">{{ <link_label> }}</a>
+```
+
+### `ibexa_checkout_step_url()`
+
+The `ibexa_checkout_step_url()` function returns a URL address of the step.
+
+``` html+twig 
+<a href="{{ ibexa_checkout_step_url(checkout, step) }}">{{ <target_page_label> }}</a>
```

Comment:
There is also a third parameter '$schemaRelative = false` by default which if specified (true/false), generates relative or absolute URL.